DataManagementSystem/DB/FileCatalogComponents/SecurityPolicies/VOMSPolicy.py

```diff
@@ -606,23 +570,22 @@ def hasAccess( self, opType, paths, credDict ):
       policyToExecute = self.__policyChangePathMode
 
     if not policyToExecute:
-      return S_ERROR( "No policy matching operation %s" % opType )
+      return S_ERROR("No policy matching operation %s" % opType)
 
-    res = policyToExecute( paths, credDict )
+    res = policyToExecute(paths, credDict)
 
     return res
 
-
-  def getPathPermissions( self, paths, credDict ):
+  def getPathPermissions(self, paths, credDict):
     """ This method is meant to disappear, hopefully soon,
         but as long as we have clients from versions < v6r14,
```

Comment:
Are there still clients before v6r14?

Reply:
I hope not, and anyway considering that this will go to at least version v7r1, I would remove this.